pkg/capacity/capacity.go

```diff
@@ -19,44 +19,43 @@ import (
 	"fmt"
 	"os"
 
-	"k8s.io/client-go/kubernetes"
-	metrics "k8s.io/metrics/pkg/client/clientset/versioned"
-
 	"github.com/robscott/kube-capacity/pkg/kube"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 	v1beta1 "k8s.io/metrics/pkg/apis/metrics/v1beta1"
+	metrics "k8s.io/metrics/pkg/client/clientset/versioned"
 )
 
 // FetchAndPrint gathers cluster resource data and outputs it
-func FetchAndPrint(showContainers, showPods, showUtil, showPodCount, excludeTainted, availableFormat bool, podLabels, nodeLabels, namespaceLabels, namespace, kubeContext, kubeConfig, output, sortBy string) {
-	clientset, err := kube.NewClientSet(kubeContext, kubeConfig)
+func FetchAndPrint(option Option) {
```

Comment:
Similar to other comment, recommend using plural names here.
```suggestion
func FetchAndPrint(opts Options) {
```